lib/components/search/search.tsx

```diff
@@ -130,6 +131,12 @@ const Search: React.FC<unknown> = () => {
         .container {
           visibility: hidden;
         }
+        .container:global(ul),
+        .container:global(li) {
```

Comment:
A space is needed before the `global`, otherwise it will affect the global style, see list in [the introduction](https://react-7b3zuxmg2-geist-ui.vercel.app/en-us/guide/introduction).
Probably can be solved after updating the remote to the latest.